modules/twinkleprod.js

```diff
@@ -172,6 +172,58 @@ Twinkle.prod.callback.prodtypechanged = function(event) {
 };
 
 Twinkle.prod.callbacks = {
+	checkpriors: function(apiobj) {
+		var xmlDoc = apiobj.responseXML;
+		var statelem = apiobj.statelem;
+		var params = apiobj.params;
+
+		// Check existence of AfD, would be easier if mw.Title.exists() worked [[phab:T184953]]
+		var exists;
+		if (namespace === 'article') {
+			var query = {
+				'action': 'query',
+				'prop': 'revisions',
+				'titles': 'Wikipedia:Articles for deletion/' + Morebits.pageNameNorm
+			};
+			var wikipedia_api = new Morebits.wiki.api('Checking for prior AfDs', query, function(apiobj) {
+				var xmlDoc = apiobj.responseXML;
+				exists = $(xmlDoc).find('rev').attr('revid');
+			});
+			// Wait for API call to finish
+			wikipedia_api.post({
+				async: false
+			});
+		}
+		if (exists) {
+			statelem.warn( 'Previous AfD for this page title found, aborting procedure' );
+			return;
+		}
+
+		// Check talk page for category indicating prior PROD
+		var cats = $(xmlDoc).find('categories');
+		if(cats.length) {
+			if(params.blp) {
+				if( !confirm( 'Do you still want to continue applying BLPPROD? ' ) ) {
```

Comment:
Should be `!confirm( 'Previous PROD nomination found on talk page. Do you still want to continue applying BLPPROD? ' )`. Right?

Reply:
Definite improvement!